Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/AssetAttribute/Breaker.tsx

```diff
@@ -26,10 +26,43 @@ import * as _ from 'lodash';
 import { Application, OpenXDA } from '@gpa-gemstone/application-typings';
 import { AssetAttributes } from './Asset';
 import { getSpareBreakersForSubstation } from '../../../TS/Services/Asset';
-import { CheckBox, Input } from '@gpa-gemstone/react-forms';
+import { CheckBox, Input, SearchableSelect } from '@gpa-gemstone/react-forms';
 import { useAppSelector } from '../hooks';
 import { SelectRoles } from '../Store/UserSettings';
 
+interface IOption { Value: string | number; Label: string }
+
+function searchSCADAPoint(search: string): [Promise<IOption[]>, () => void] {
+    let searchHandle: JQuery.jqXHR<IOption[]>;
+    const searchPromise = new Promise<IOption[]>((resolve, reject) => {
```

Comment:
This seems redundant.
a `jqXhr` is technically a `Promise` let me see if we have an example where we deal with this.